projects/multiselect/src/lib/options/options.component.spec.ts

```diff
@@ -16,59 +25,112 @@ describe('OptionsComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(OptionsComponent);
     component = fixture.componentInstance;
+    debugElement = fixture.debugElement;
+    options = [
+      { "id": 1, "name": "Test 1"},
+      { "id": 2, "name": "Test 2"},
+      { "id": 3, "name": "Test 3"},
+      { "id": 4, "name": "Test 4"},
+      { "id": 5, "name": "Test 5"},
+      { "id": 6, "name": "Test 6", disabled: true }
+    ];
+    component.options = [...options];
+    multiselect = new NgxMultiselectComponent(<ElementRef<any>>null, new NgxMultiselectService());
+    multiselect.multiple = true;
+    multiselect.setOptions(options);
+    component.selectOption.subscribe(option => {
+      clickedOption = option;
+      multiselect.select(option);
+      component.options = multiselect.getOptions();
+      fixture.detectChanges();
+    });
+    multiselectSelectSpy = spyOn(multiselect, 'select').and.callThrough();
     fixture.detectChanges();
   });
 
   it('Component should gets added into the DOM', () => {
     // arrange
     // act
     // assert
-    expect(true).toBeTruthy();
+    expect(component).toBeDefined();
   });
 
   describe('Option', () => {
     it('it should emit an event to parent component to select underlying option', () => {
       // arrange
+      const optionsElements = debugElement.queryAll(By.css('.option'));
       // act
+      optionsElements[1].triggerEventHandler('click', {});
+      fixture.detectChanges();
       // assert
-      expect(true).toBeTruthy();
+      const markedOptions = debugElement.queryAll(By.css('.option.marked'));
+      expect(markedOptions.length).toBe(1);
+      expect(clickedOption.ticked).toBe(true);
+      expect(multiselectSelectSpy).toHaveBeenCalled();
+      expect(multiselectSelectSpy).toHaveBeenCalledWith(clickedOption);
     });
     it('should select an option on click of selected option', () => {
       // arrange
+      const optionsElements = debugElement.queryAll(By.css('.option'));
       // act
+      optionsElements[1].triggerEventHandler('click', {});
+      fixture.detectChanges();
       // assert
-      expect(true).toBeTruthy();
+      const markedOptions = debugElement.queryAll(By.css('.option.marked'));
+      expect(markedOptions.length).toBe(1);
+      expect(clickedOption.ticked).toBe(true);
+      expect(multiselectSelectSpy).toHaveBeenCalled();
+      expect(multiselectSelectSpy).toHaveBeenCalledWith(clickedOption);
     });
     it('should de-select an option on click of selected option', () => {
       // arrange
+      const optionsElements = debugElement.queryAll(By.css('.option'));
+      optionsElements[1].triggerEventHandler('click', {});
+      fixture.detectChanges();
       // act
+      optionsElements[1].triggerEventHandler('click', {});
```

Comment:
You can emit null event 
`optionsElements[1].triggerEventHandler('click', null);`